preso-ro/speakers.yml

```diff
@@ -0,0 +1,13 @@
+---
+-
+  :name: "Sameera Gayan"
```

Comment:
This part might seem a little labourious but it would help out with repeat speakers.